bundles/org.openhab.binding.solax/src/main/resources/OH-INF/i18n/solax.properties

```diff
@@ -0,0 +1,149 @@
+# binding name
+
+bindingName = Solax Inverter Binding
```

Comment:
This should not be here. When using the command `mvn i18n:generate-default-translations` something like this should be automatically generated at the top of the file:
```ini
# add-on

addon.solax.name = Solax Binding
addon.solax.description = This is the binding for Solax inverters.
```

Also, it seems like there are quite some left-overs from previous channel names etc. Perhaps it would be easier to delete the file and generate it from scratch, and then manually add your custom text:

```ini
offline.communication-error.json-cannot-be-retrieved=JSON data could not be retrieved.
```